File: pset6/mario.py
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if 9 > h > 0 :
        break

for r in range(1, h + 1):
    print(" " * (h - r) + "#" * r + " " * 2 + "#" * r )

File: pset1/hello/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Height of Pyramid:\n");
    }
    while(h < 0 || h >= 9);

    for(int row = 0; row < h; row++)
    {
        for(int i = 0; i < row; i++)
    
        {
            printf("#");
        }
    }
}